README.md

```diff
@@ -68,7 +70,6 @@ const App = (props) => {
 ### `withMedia`
 
 This is a HOC to provide media match props to your component.
-This abstracts away context so that if there is any changes to the API in the future its easier to upgrade (see: [React Context](https://facebook.github.io/react/docs/context.html))
```

Comment:
why was this removed?

Reply:
Didn't think it was relevant now that React 16's Context API is not experimental.

> React previously shipped with an experimental context API

https://reactjs.org/docs/context.html#legacy-api

We should use the legacy one while we support React 15, I'll mention this.